### ISSUE TITLE
amp-form: support enctype=application/x-www-form-urlencoded for action-xhr

### ISSUE THREAD

**Issue by pwdonald** [ampproject/amphtml#7607]
When using amp-form with `method="POST`" and `action-xhr` the resulting request is `multipart/form-data` binary. I tried setting `enctype `to `application/x-www-form-urlencoded` but it was ignored. 

Is this by design? If so I do not see it documented.

**Comment by aghassemi** [comment]
@pwdonald Currently not supported, but something we may support later. I will keep your issue open as a feature request and will update the documentation to reflect the lack of support.

**Comment by rudygalfi** [comment]
@aghassemi Is there anything special to know about supporting this, or does it just need prioritization guidance?

**Comment by aghassemi** [comment]
@rudygalfi no, just needs prioritization. Ideally goes in the same bucket as https://github.com/ampproject/amphtml/issues/7695

**Comment by g-guth** [comment]
Any news on that?

**Comment by philkrie** [comment]
Bumping this, get this request a lot. Seems like something that should be supported, especially since the amp-form documentation states that "All other form attributes are optional.", without explaining that some just don't work. That's probably caused a lot of frustration.

**Comment by rudygalfi** [comment]
cc @nainar Can you take a fresh look at prioritization?

**Comment by mcmd** [comment]
Hi, just ran into this today. Any updates on prioritization? Thanks!

**Comment by philkrie** [comment]
+1, at the very least the documentation should be updated ASAP to specify that 1. not all form attributes are supported, and 2. enctype specifically is not configurable. This will save a lot of headache

**Comment by jerimiahwelch** [comment]
+1. For almost every project we do, we end up a modifying server endpoints that only accept `application/x-www-form-urlencoded` to `accept multipart/form-data` as well. Supporting `application/x-www-form-urlencoded` in amp-form would lower the level of effort to  support AMP in many ecommerce environments.

**Comment by caroqliu** [comment]
@krdwan will be working on getting a fix for this soon :)
